Add more detailed docs on containerized firefox install support

micimize opened the pull request:
Snap & Flatpak now have support for native messaging so I wrote some docs to try and help resolve https://github.com/tridactyl/tridactyl/issues/2406.

This maybe belongs in a Wiki page instead of the main readme – lmk and I'll move it (or someone else can, w/e).

**NOTE for KDE users:** [Running firefox from krunner doesn't work](https://discourse.ubuntu.com/t/call-for-testing-native-messaging-support-in-the-firefox-snap/29759/102?u=micimize), you have to test from command line, or run with `sh -c 'firefox'`. Not sure why

bovine3dom commented:
Looks good, thanks! Could you maybe add a link to it on src/excmds.ts in the native() help?

micimize commented:
Add links in both `native` and `nativeinstall` helps and cleaned up the copy some. Also added a link to this PR as a makeshift troubleshooting thread as the note i just discovered/added about krunner doesn't seem appropriate to add to the main documentat.

micimize commented on src/excmds.ts:
```diff
@@ -738,6 +739,8 @@ export async function native() {
  * The native messenger's source code may be found here: https://github.com/tridactyl/native_messenger/blob/master/src/native_main.nim
  *
  * If your corporate IT policy disallows execution of binaries which have not been whitelisted but allows Python scripts, you may instead use the old native messenger by running `install.sh` or `win_install.ps1` from https://github.com/tridactyl/tridactyl/tree/master/native - the main downside is that it is significantly slower.
+ * 
```
```suggestion
 *
```

Rahlir commented:
I think there is a typo. The command should be `flatpak permission-set...` not `flatpack`.
